### ISSUE TITLE
Map should be keyboard navigable

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7190]
**CKAN version**
https://demo.ckan.org/dataset/sample-dataset-1/resource/ecd4a62d-998b-46e4-8a64-cadac2125c64

**Describe the bug**
There is no reason that you should need to be able to use a mouse to navigate this map. 

You can navigate to the button with the tab key, but then can't activate it.

The arrow keys left/right work but not up/down.

**Steps to reproduce**
- Navigate to the URL above.
- Without using your mouse or touchpad, using the tab key, navigate to the map.
- try to zoom in/out into the map

**Expected behavior**
You should be able to use the button to zoom in & out.  It would also be useful if this were tested with voice control devices. Not every can use a mouse. 

**Additional details**
https://webaim.org/techniques/keyboard/
